gas/testsuite/gas/riscv/zc-zca-cm-lh-fail-march.d

```diff
@@ -0,0 +1,3 @@
+#as: -march=rv64ifd_zca
+#source: zc-zca-cm-lh-fail-march.s
+#error_output: zc-zca-cm-lh-fail-march.l
```

Comment:
No newline at end of file.

(I'll stop commenting on these down, but check all these)